dsf-gdb/org.eclipse.cdt.tests.dsf.gdb/src/org/eclipse/cdt/tests/dsf/gdb/framework/BaseTestCase.java

```diff
@@ -680,7 +684,17 @@ private void launchGdbServer() throws Exception {
 				String server = (String) launchAttributes.get(ATTR_DEBUG_SERVER_NAME);
 				String port = (String) launchAttributes.get(IGDBLaunchConfigurationConstants.ATTR_PORT);
 				String program = (String) launchAttributes.get(ICDTLaunchConfigurationConstants.ATTR_PROGRAM_NAME);
-				String[] commandLine = { server, ":" + port, program };
+				boolean multi = Boolean.valueOf((String) launchAttributes.get(ITestConstants.LAUNCH_GDB_SERVER_MULTI));
+				boolean noprogram = Boolean
+						.valueOf((String) launchAttributes.get(ITestConstants.LAUNCH_GDB_SERVER_WITHOUT_PROGRAM));
+				List<String> commandArgs = new ArrayList<>();
+				commandArgs.add(server);
+				if (multi)
```

Comment:
Can you add curly bracers around the single line statements to make a clear scope?
I know it's not mandatory, but most of the code in CDT does use it even when it's not mandatory to make the intended scope clearer.
This comment applies to other places in the PR too.